desktop/npm-helper.js

```diff
@@ -333,26 +341,135 @@ function undiff () {
   console.log('Success! Wrote ./log.json')
 }
 
-function updatedFonts () {
-  const project = JSON.parse(fs.readFileSync('./shared/images/iconfont/kb-icomoon-project.json', 'utf8'))
-  const fontPrefix = 'kb-iconfont-'
-  const icons = {}
+function svgToGridMap () {
+  const grids = {}
+
+  fs.readdirSync('../shared/images/iconfont').forEach(i => {
+    const match = i.match(/^kb-iconfont-(.*)-(\d+).svg$/)
+    if (match && match.length === 3) {
+      const name = match[1]
+      const p = path.resolve('../shared/images/iconfont', i)
+      const gridSize = match[2]
 
-  project.iconSets.forEach(set => {
-    set.icons.forEach(icon => {
-      const name = icon.tags[0]
-      if (name.startsWith(fontPrefix)) {
-        const shortName = name.slice(3)
-        icons[shortName] = {
-          isFont: true,
-          gridSize: icon.grid,
-        }
-      } else {
-        console.log('Invalid icon font name!: ', name)
+      if (!grids[gridSize]) {
+        grids[gridSize] = {}
       }
-    })
+
+      grids[gridSize][name] = {name, gridSize, path: p}
+    }
   })
 
+  return grids
+}
+function generateIcoMoon () {
+  const svgPaths = {}
+  // Need to get the svg info from iconmoon. Couldn't figure out how to derive exactly what they need from the files themselves
+  JSON.parse(fs.readFileSync('../shared/images/iconfont/kb-icomoon-project-app.json', 'utf8')).icons.forEach(icon => {
```

Comment:
The project file actually contains the svg data. Icomoon needs this in a very specific form (vs reading the svg files directly). So we have to submit the icons to icomoon just to get the svg data. This is the code where we merge this into our own data